test/test.sh

```diff
@@ -986,6 +986,28 @@ test_shadow_password_server_login() {
 	echo "curuser=$curuser"
 	test "$curuser" = "shadowuser2" || return 1
 
+	# user defined in ini [users] section with good recently changed password from PostgreSQL server
```

Comment:
More test cases to ensure that users can still login with new password when it's changed at Postgres.